### ISSUE TITLE
TypeScript Error: Not a module

### ISSUE THREAD

**Issue by roythuis** [Mitcheljager/svelte-tiny-slider#2]
When importing TinySlider like the installation guide says with `import { TinySlider } from "svelte-tiny-slider"` I get the following TypeScript error message in VS Code:

> "File '{{PATH}}/node_modules/svelte-tiny-slider/dist/index.d.ts' is not a module.ts(2306)"

**Comment by Mitcheljager** [comment]
What version of Svelte are you using?

**Comment by roythuis** [comment]
Hi Mitchel,

I just upgraded to Svelte 4 before installing this package. Here are my devDependencies:
```
"devDependencies": {
		"@sveltejs/adapter-auto": "^2.0.0",
		"@sveltejs/kit": "^1.20.4",
		"@typescript-eslint/eslint-plugin": "^5.45.0",
		"@typescript-eslint/parser": "^5.45.0",
		"eslint": "^8.28.0",
		"eslint-plugin-svelte": "^2.30.0",
		"sass": "^1.62.1",
		"svelte": "^4.0.0",
		"svelte-check": "^3.4.3",
		"svelte-preprocess": "^5.0.3",
		"tslib": "^2.4.1",
		"typescript": "^5.0.0",
		"vite": "^4.3.0"
	},
```

**Comment by Mitcheljager** [comment]
This should be fixed with 1.1.4. Thanks!
